scripts/o3de/ExportScripts/export_source_built_project.py

```diff
@@ -0,0 +1,221 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+import argparse
+import logging
+import sys
+
+import o3de.export_project as exp
+import pathlib
+
+from typing import List
+
+
+def export_standalone_monolithic_project_centric(ctx: exp.O3DEScriptExportContext,
```

Comment:
There are some parts of the function logic which checks for engine_centric build in the context. Maybe we should strip `project_centric` from the function name?